config/jobs/gke/gke-staging-containerd.yaml

```diff
@@ -0,0 +1,243 @@
+periodics:
+- interval: 1h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-cos-containerd-gke
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - args:
+      - --timeout=70
+      - --bare
+      - --
+      - --check-leaked-resources
+      - --cluster=
+      - --deployment=gke
+      - --extract=ci/latest
+      - --gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging
+      - --gcp-node-image=cos_containerd
+      - --gcp-zone=us-central1-f
+      - --ginkgo-parallel
+      - --gke-create-command=container clusters create --quiet --enable-kubernetes-alpha
+      - --gke-environment=test
+      - --provider=gke
+      - --test_args=--ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8
```

Comment:
so in yaml, we don't need to escape backslashes. see https://github.com/kubernetes/test-infra/pull/8712

Reply:
Done